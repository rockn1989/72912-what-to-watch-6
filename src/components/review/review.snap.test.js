import React from 'react';
import {render} from '@testing-library/react';
import Review from './review';

it(`Should Review render correctly`, () => {
  const data = {
    text: `test`,
    date: `12/03/1999`,
    author: `Alex`,
    rating: 15
  };

  const {container} = render(<Review {...data} />);
  expect(container).toMatchSnapshot();
});
