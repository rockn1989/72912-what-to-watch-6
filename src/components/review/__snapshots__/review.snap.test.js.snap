// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<div>
  <div
    class="review"
  >
    <blockquote
      class="review__quote"
    >
      <p
        class="review__text"
      >
        test
      </p>
      <footer
        class="review__details"
      >
        <cite
          class="review__author"
        >
          Alex
        </cite>
        <time
          class="review__date"
          dateime="2016-12-24"
        >
          12/03/1999
        </time>
      </footer>
    </blockquote>
    <div
      class="review__rating"
    >
      15
    </div>
  </div>
</div>
`;
